revenue_app/templates/revenue_app/query.html

```diff
@@ -64,9 +64,25 @@ <h1>Queries</h1>
   {% csrf_token %}
   <div class="row mt-3">
     <div class="col-12">
-      <button type="submit" class="btn btn-block btn-evb-orange">Make Query</button>
+      <button id="submit-btn" type="submit" class="btn btn-block btn-evb-orange">Make Query</button>
     </div>
   </div>
 </form>
+<div style="display: flex; justify-content: center;">
```

Comment:
Please make this a .css class. In-line styling is pretty bad for reusability.

Reply:
`<div class="d-flex justify-content-center">...</div>`

Reply:
Resolved.